internal/snips/file.go

```diff
@@ -58,3 +61,33 @@ func (f *File) Visibility() string {
 
 	return "public"
 }
+
+var decoder, _ = zstd.NewReader(nil)
```

Comment:
I was looking at the zstd docs, this is an interesting pattern that it uses!

Normally I'd think to use it within the methods during the lifetime of the content, but clever that it'll actually [concurrency safe](https://github.com/klauspost/compress/tree/master/zstd#usage-1) from multiple callers.

However we should be careful, the `decoder.Close()` a few lines below will actually dispose an the reader and prevent it any decoder goroutines from being created.

I think it'd actually prefer to put these `NewReader`/`NewWriter` _within_ the getter/setters themselves. That way we don't need to worry about tweaking some options like the following:

> It will only allow a certain number of concurrent operations to run. To tweak that yourself use the `WithDecoderConcurrency(n)` option when creating the decoder.

I don't think the perf gain from having these outside the methods will really be noticeable to snips 😄